endoreg_db/services/video_import.py

```
@@ -192,7 +191,197 @@ def import_and_anonymize(
     if frame_cleaning_available and video_file_obj.raw_file:
+        try:
+            logger.info("Starting frame-level anonymization with processor ROI masking...")
+            
+            # Get processor ROI information for masking
+            processor_roi = None
+            endoscope_roi = None
+            
+            try:
+                if video_file_obj.video_meta and video_file_obj.video_meta.processor:
+                    processor = video_file_obj.video_meta.processor
+                    video_meta = video_file_obj.video_meta
```

Ruff (0.12.2)

Local variable `video_meta` is assigned to but never used

Remove assignment to unused variable `video_meta`

(F841)